src/test/java/org/dinosaur/foodbowl/domain/store/docs/StoreControllerDocsTest.java

```diff
@@ -0,0 +1,184 @@
+import static org.springframework.restdocs.payload.PayloadDocumentation.requestFields;
+import static org.springframework.restdocs.payload.PayloadDocumentation.responseFields;
+import static org.springframework.restdocs.request.RequestDocumentation.parameterWithName;
+import static org.springframework.restdocs.request.RequestDocumentation.queryParameters;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
+import org.dinosaur.foodbowl.MockApiTest;
+import org.dinosaur.foodbowl.domain.member.entity.Role.RoleType;
+import org.dinosaur.foodbowl.domain.store.api.StoreController;
+import org.dinosaur.foodbowl.domain.store.application.StoreService;
+import org.dinosaur.foodbowl.domain.store.dto.StoreRequest;
+import org.dinosaur.foodbowl.domain.store.dto.StoreResponse;
+import org.dinosaur.foodbowl.global.config.security.jwt.JwtTokenProvider;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.restdocs.headers.HeaderDescriptor;
+import org.springframework.restdocs.payload.FieldDescriptor;
+import org.springframework.restdocs.payload.JsonFieldType;
+
+@WebMvcTest(StoreController.class)
+public class StoreControllerDocsTest extends MockApiTest {
+
+    @MockBean
+    private StoreService storeService;
+
+    @Autowired
+    private JwtTokenProvider jwtTokenProvider;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Test
+    @DisplayName("가게 조회 응답을 반환한다.")
+    void findSuccess() throws Exception {
+        String address = "서울시+송파구+올림픽로+473";
+        StoreResponse storeResponse = createResponse();
+        given(storeService.findByAddress(any(String.class))).willReturn(storeResponse);
+
+        mockMvc.perform(get("/api/v1/stores")
+                        .queryParam("address", address)
+                        .header("Authorization", "Bearer " + jwtTokenProvider.createAccessToken(1L, RoleType.ROLE_회원))
+                        .characterEncoding(StandardCharsets.UTF_8))
+                .andExpect(status().isOk())
+                .andDo(print())
+                .andDo(document("api-v1-stores-read",
+                        requestHeaders(
+                                headerWithName("Authorization").description("Foodbowl 인증 토큰")
```

Comment:
```suggestion
                                headerWithName("Authorization").description("서버에서 발급한 인증 토큰")
```
이전까지는 위와 같이 사용했던 것 같은데 통일있게 맞추는게 좋을 것 같아요 :)